runtime/jcl/common/java_lang_J9VMInternals.cpp

```diff
@@ -32,14 +32,17 @@
 extern "C" {
 
 jobject JNICALL
-Java_java_lang_J9VMInternals_getStackTrace(JNIEnv * env, jclass recv, jobject throwable, jboolean pruneConstructors)
+Java_java_lang_J9VMInternals_getStackTrace(JNIEnv *env, jclass recv, jobject throwable, jboolean pruneConstructors)
 {
-	J9VMThread *currentThread = (J9VMThread*)env;
+	if (NULL == throwable) {
+		return NULL;
+	}
```

Comment:
Please put this under Java 19 ifdef, for pre-Java 19, it is not possible for a null throwable to be passed into this method.

Reply:
done